repos.d/apkfission.yaml

```diff
@@ -0,0 +1,40 @@
+###########################################################################
+# APK Fission packages
+#
+# https://apkfission.net/install.html
+###########################################################################
+
+{% macro apkfission(version, subrepos=['main','nonfree'], archs=['aarch64','armv7','pmmx','ppc','ppc64','x86_64']) %}
```

Comment:
This probably belongs to adelie.yaml instead a dedicated file

Reply:
APK Fission is similar to, and named somewhat after, RPM Fusion – which has its own file despite being only for Fedora.  It has no organisational connection to Adélie.